apps/menu-bar/src/components/Button.tsx

```diff
@@ -14,10 +14,9 @@ import {useCurrentTheme} from '../utils/useExpoTheme';
 import {addOpacity} from '../utils/theme';
 
 type Color = 'default' | 'primary';
-interface Props extends TouchableOpacityProps {
-  children: string;
```

Comment:
we should probably keep this, otherwise, people may try to pass things other than strings

Reply:
Then we should not use `children`, and go with dedicated prop (as native Button does).